contentcuration/static/admin/css/base.css

```diff
@@ -0,0 +1,850 @@
+/*
```

Comment:
There should be no committed files in this 'static' folder.

Let's sit down, fix the .gitignore file so that it ignores this folder, and we can amend your commit history to make sure these files were never included in the first place (as far as git is concerned).